doc/intro.md

```diff
@@ -84,6 +81,23 @@ Three alternatives are available to produce population projections:
 
```

Comment:
It's not clear what these alternatives really are (e.g. alternative data inputs, alternative calculations, etc). I see 3 steps, but no matter what input data you use in the first step (the historic stuff) don't you always need to calc future rates and do the forward projection? What exactly is trying to be conveyed here?